internal/config/template/testdata/external_dirs/infracost.yml.tmpl

```diff
@@ -0,0 +1,17 @@
+{{- if pathExists "../different_env_dirs" "main.tf" }}
+external_dir: fail
+{{- else }}
+external_dir: success
+{{- end }}
+
+{{- if pathExists "." "main.tf" }}
+dot_path: success
+{{- else }}
+dot_path: fail
+{{- end }}
+
+{{- if pathExists "../external_dirs" "main.tf" }}
+dot_dot_path: success
+{{- else }}
+dot_dot_path: fail
+{{- end }}
```

Comment:
This test doesn't actually seem to be testing what we're adding which is scoping to the repo directory. Can we add a few test cases to make sure that's covered, i.e. paths outside the repo directory.

Reply:
Isn't that what the first case on line 1 is testing?

Reply:
but shouldn't that fail? The golden file output shows it passed?

Reply:
ah sorry I'm reading the if else wrong - that was confusing me. Can we change the statements around so the output is "fail"?

Reply:
👍 done